src/equations/ideal_glm_mhd_1d.jl

```diff
@@ -277,6 +277,22 @@ end
     λ_max = max(abs(v_ll), abs(v_rr)) + max(cf_ll, cf_rr)
 end
 
+# Calculate estimates for minimum and maximum wave speeds for HLL-type fluxes
+@inline function min_max_speed_naive(u_ll, u_rr, orientation::Integer,
+                                     equations::IdealGlmMhdEquations1D)
+    rho_ll, rho_v1_ll, _ = u_ll
+    rho_rr, rho_v1_rr, _ = u_rr
+
+    # Calculate primitive variables
+    v1_ll = rho_v1_ll / rho_ll
+    v1_rr = rho_v1_rr / rho_rr
+
+    λ_min = v1_ll - calc_fast_wavespeed(u_ll, orientation, equations)
+    λ_max = v1_rr + calc_fast_wavespeed(u_rr, orientation, equations)
+
+    return λ_min, λ_max
+end
+
 # More refined estimates for minimum and maximum wave speeds for HLL-type fluxes
 @inline function min_max_speed_davis(u_ll, u_rr, orientation::Integer,
```

Comment:
Same here, update comment? The Davis wave speeds are no longer used for the HLL flux right? The Einfeldt estimates compare the naive speeds to the Roe estimates as far as I recall.

Reply:
So Davis is what I have been told as the "classical" HLL scheme, that is why we have it here.

Reply:
Okay, I think I am understanding things now. So within `numerical_fluxes.jl` is it that the generic functions are set with the style `flux_hll = FluxHLL(min_max_speed_davis)` and `flux_hlle = FluxHLL(min_max_speed_einfeldt`? If this is the case I am still confused what exactly `min_max_speed_naive` is used for. Is it for LLF?

Reply:
I think `min_max_speed_naive` was the name I introduced to basically mean something like `min_max_speed_davis`. Please correct me if I don't remember this correctly, @DanielDoehring. If so, we should consider removing `min_max_speed_naive` and just use `min_max_speed_davis`.

Reply:
Yes, the `naive` estimate is similar to the classic one by Davis, but is (as far as I know) seldom taught or used in practice.

However, this is still the default wave speed estimate:

https://github.com/trixi-framework/Trixi.jl/blob/3186df5be024f1253d282339058d7df3e1a02307/src/equations/numerical_fluxes.jl#L226

https://github.com/trixi-framework/Trixi.jl/blob/3186df5be024f1253d282339058d7df3e1a02307/src/equations/numerical_fluxes.jl#L300-L305

Now would be the time to change this default behaviour, though.

Reply:
Do we have any case where the naive version is different from a classical Davis estimate?

Reply:
You mean an elixir where e.g. the errors would be different?

Reply:
Where the implementation of `_naive` and `_davis` is different

Reply:
They are different for all (SWE, CEE, MHD) equations, as the `naive` version is somewhat biased towards left and right state, whereas the `davis` one is not:

https://github.com/trixi-framework/Trixi.jl/blob/276dc3ca22a4c72b81aa6659e57ff1382dd1ae0c/src/equations/compressible_euler_1d.jl#L702-L727

Reply:
I see, thanks!